Week-2/q1/public/css/index.css

```diff
@@ -0,0 +1,201 @@
+@import url(https://fonts.googleapis.com/css2?family=Lato);
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body,
+html {
+    overflow-x: hidden;
+    font-family: 'Lato';
+}
+
+a {
+    text-decoration: none;
+    color: #43689b;
+}
+
+
+/* Header */
+header {
+    text-align: center;
+    align-items: center;
+    margin-bottom: 5px;
+    padding: 15px;
+}
+
+.header {
+    color: #3D4149;
```

Comment:
Try using CSS variables for these color codes. like Header Background color et c..